src/util/LazyJsonParser.cpp

```diff
@@ -85,8 +78,8 @@ void LazyJsonParser::parseLiteral(size_t& idx) {
       case '"':
         // End of literal.
         if (std::holds_alternative<BeforeArrayPath>(state_)) {
-          std::get<BeforeArrayPath>(state_).litLength_ =
-              idx - std::get<BeforeArrayPath>(state_).litStart_;
+          std::get<BeforeArrayPath>(state_).optLiteral_->length_ =
+              idx - std::get<BeforeArrayPath>(state_).optLiteral_->start_;
```

Comment:
use ` optLiteral.value().lent_` , some for ` start` .
These unsafe accesses give me goosebumps.